src/backend/distributed/citus--7.1-2--7.1-3.sql

```diff
@@ -0,0 +1,14 @@
+/* citus--7.1-2--7.1-3 */
+
+CREATE TABLE citus.pg_dist_node_metadata(
+    metadata jsonb
```

Comment:
Using `jsonb` instead of `json` since it provides better API. Although it makes no difference in this patch.

Reply:
Could use a `NOT NULL` here…